src/Stryker.Core/Stryker.Core/TestRunners/VsTest/RunEventHandler.cs

```diff
@@ -40,38 +40,38 @@ public bool IsComplete()
 
         public TestResult Result()
         {
-            var result = _results.Aggregate((TestResult) null, (acc, next) =>
-            {
-                if (acc == null)
-                {
-                    return next;
-                }
-                if (next.Outcome == TestOutcome.Failed || acc.Outcome == TestOutcome.None)
-                {
-                    acc.Outcome = next.Outcome;
-                }
-                if (acc.StartTime > next.StartTime)
-                {
-                    acc.StartTime = next.StartTime;
-                }
-                if (acc.EndTime < next.EndTime)
-                {
-                    acc.EndTime = next.EndTime;
-                }
-
-                foreach (var message in next.Messages)
-                {
-                    acc.Messages.Add(message);
-                }
-
-                acc.Duration = acc.EndTime - acc.StartTime;
-                return acc;
-            });
+            var result = _results.Aggregate((TestResult)null, (acc, next) =>
```

Comment:
Something is off about the indenting change here

Reply:
Yes, something was off. It's fixed now.